src/kite.js

```diff
@@ -231,8 +215,32 @@ const Kite = {
     );
 
     this.disposables.push(
-      vscode.commands.registerTextEditorCommand("kite.related-files", (textEditor) => {
-        opn(`http://localhost:46624/codenav/related/${encodeURIComponent(textEditor.document.fileName)}`);
+      vscode.commands.registerTextEditorCommand("kite.related-code-from-file", (textEditor) => {
+        KiteAPI
+          .requestRelatedCode(textEditor.document.fileName, null, null, "vscode")
```

Comment:
This if more for KiteAPI, but stylistically, I think we should put non-optional args before any optional args.